src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -14,14 +14,14 @@
     defined(TARGET_R9M_TX)        || \
     defined(TARGET_TX_ES915TX)    || \
     defined(TARGET_ES900TX)
-#if defined(UNLOCK_HIGHER_POWER) || defined(TARGET_NAMIMNORC_TX)
+#if defined(UNLOCK_HIGHER_POWER) || defined(HAS_BUILT_IN_FAN)
```

Comment:
you should remove `HAS_BUILT_IN_FAN` and set `UNLOCK_HIGHER_POWER` for NamimnoRC targets. 
There could be a target which has a FAN but not capable for higher power :)

Reply:
Good point. Will do.